specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,126 @@
+|                           | Streaming Analytics               | Processing of real-time data streams to detect patterns, trends, and anomalies as they occur.                                                                                                                                              |
+|                           | Data Processing                   | Collection, transformation, and organization of data to make it useful for analysis. Includes tasks like data cleaning, integration, and aggregation.                                                                                      |
+|                           | Other (Analytics)                 | Analytics services that do not fall into one of the defined subcategories.                                                                                                                                                                 |
+|                           |                                   |                                                                                                                                                                                                                                            |
+| Business Applications     | Other (Business Applications)     | All Business Applications services.                                                                                                                                                                                                        |
+|                           |                                   |                                                                                                                                                                                                                                            |
+| Compute                   | Virtual Machines                  | Deployment and running of various applications in a flexible computing environment.                                                                                                                                                        |
+|                           | End User Computing                | Systems and tools that allow non-IT users to create and manage their own applications and data, including desktop virtualization, mobile computing, and self-service analytics.                                                            |
+|                           | Containers                        | Lightweight, portable units that package applications and their dependencies, ensuring they run consistently across different environments.                                                                                                |
```

Comment:
Shouldn't this be Container Orchestration? Containers seems a bit misleading given that what providers capture in this today is just the orchestration cost?

Reply:
I guess I hadn't thought of it that way... but I went with this shorter name, given that all three major CSPs capture these costs under a Category that they call `Containers`.  I will raise the topic in tomorrow's TF1 call and see what the team thinks.

Reply:
I get that. I meant more for the description. If I read that description, I would expect to find my cost of containers to be in this category when I'm not going to find the bulk of it there. I get that the name may need to be Containers just because everyone has done that same thing (even though its not accurate). At least in the description, we could try to be accurate.

Reply:
Very fair point.  Updated via a separate commit to read:

> Automated management of containerized applications.

Reply:
Resolving this comment.  Please unresolve if the above change doesn't work.